src/os/bluestore/BlueStore.cc

```diff
@@ -3866,6 +3872,28 @@ void BlueStore::MempoolThread::_resize_shards(bool interval_stats)
   }
 }
 
+void BlueStore::MempoolThread::_update_cache_settings()
+{
+  auto cct = store->cct;
+  uint64_t target = store->osd_memory_target;
+  uint64_t base = store->osd_memory_base;
```

Comment:
Suggest to implement run-time change notifications for other config settings used here. It seems to be quite trivial to add...

Reply:
I have implemented run-time changes for the following other config settings,
- osd_memory_base
 - osd_memory_cache_min
 - osd_memory_expected_fragmentation
 - osd_memory_target_cgroup_limit_ratio

Apart from osd_memory_target, the other options are classified as either 'advanced' or 'dev' and I think should be changed with guidance.